src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.cs

```diff
@@ -196,7 +196,7 @@ private static bool IsRenameValid(ConflictResolution conflictResolution, ISymbol
                     if (semanticFactsService.SupportsParameterizedProperties)
                     {
                         otherThingsNamedTheSameExcludeMethodAndParameterizedProperty = otherThingsNamedTheSame
-                            .Where(s=> !s.MatchesKind(SymbolKind.Method, SymbolKind.Property) ||
+                            .Where(s => !s.MatchesKind(SymbolKind.Method, SymbolKind.Property) ||
```

Comment:
:grey_exclamation: This file should not be modified in this commit. It is a formatting-only change mixed in with a refactoring that does not need to touch this file.

:memo: The IDE0055 warning is preferable to mixing formatting changes into unrelated refactorings.